web/modules/custom/portland/portland.module

```diff
@@ -824,6 +824,17 @@ function _validate_document(&$form, \Drupal\Core\Form\FormStateInterface $form_s
       $efiles_link = $form_state->getValue('field_efiles_link');
       $document = $form_state->getValue('field_document');
     }
+
+    // PGOV-809 Validate document and efiles link fields
+    if( ( empty($efiles_link[0]["uri"]) && empty($document[0]["fids"]) ) ) {
+      $form_state->setErrorByName('field_document', t('Must have either an uploaded document or an eFiles link'));
+      return;
+    }
+    else if ( ( !empty($efiles_link[0]["uri"]) && !empty($document[0]["fids"]) ) ) {
```

Comment:
Again extra set of parentheses for the else if.

Reply:
Extra parentheses removed.